other/nightly/upgrade.sh

```
@@ -14,20 +15,39 @@ curl -fsSL $CDN/docker-compose.yml -o /data/coolify/source/docker-compose.yml
 curl -fsSL $CDN/docker-compose.prod.yml -o /data/coolify/source/docker-compose.prod.yml
 curl -fsSL $CDN/.env.production -o /data/coolify/source/.env.production
 
-# Merge .env and .env.production. New values will be added to .env
-awk -F '=' '!seen[$1]++' /data/coolify/source/.env /data/coolify/source/.env.production >/data/coolify/source/.env.tmp && mv /data/coolify/source/.env.tmp /data/coolify/source/.env
-# Check if PUSHER_APP_ID or PUSHER_APP_KEY or PUSHER_APP_SECRET is empty in /data/coolify/source/.env
-if grep -q "PUSHER_APP_ID=$" /data/coolify/source/.env; then
-    sed -i "s|PUSHER_APP_ID=.*|PUSHER_APP_ID=$(openssl rand -hex 32)|g" /data/coolify/source/.env
+# Backup existing .env file before making any changes
+if [ "$SKIP_BACKUP" != "true" ]; then
+    if [ -f "$ENV_FILE" ]; then
+        echo "Creating backup of existing .env file to .env-$DATE" >>$LOGFILE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)